### PR TITLE
Update companies.rst

### PR DESCRIPTION
**Pull request by ashtongreen**
Added notes on how Working Works.

### CODE REVIEW

File: doc/source/basicgameplay/companies.rst [hunk 1]
```diff
@@ -7,6 +7,16 @@ these companies, you can apply for jobs.
 
 Working a job lets you earn money, experience, and reputation with that company.
 
+While working for a company, you can click "Do something else simultaneously" be able
```
Comment (ApamNapat) [inline doc/source/basicgameplay/companies.rst:10]:
```suggestion
While working for a company, you can click "Do something else simultaneously" to be able 
```

File: doc/source/basicgameplay/companies.rst [hunk 2]
```diff
@@ -7,6 +7,16 @@ these companies, you can apply for jobs.
 
 Working a job lets you earn money, experience, and reputation with that company.
 
+While working for a company, you can click "Do something else simultaneously" be able 
+to do things while you continue to work in the background. There is a 20% penalty the
```
Comment (ApamNapat) [inline doc/source/basicgameplay/companies.rst:11]:
```suggestion
to do things while you continue to work in the background. There is a 20% penalty to the 
```

File: doc/source/basicgameplay/companies.rst [hunk 3]
```diff
@@ -7,6 +7,16 @@ these companies, you can apply for jobs.
...
 
+While working for a company, you can click "Do something else simultaneously" be able 
+to do things while you continue to work in the background. There is a 20% penalty the 
+related gains. Clicking the "Focus" button under the overview will return you to the 
+current work.   
+
+Reputation is required to apply for a promotion. This reputation is not counted towards
+your career until the shift ends, either due to the time spent or clicking the 
+"Stop Working" button. For most jobs there is a penalty of 50% of the reputation gained
+if you stop your shift early.
```
Comment (ApamNapat) [inline doc/source/basicgameplay/companies.rst:18]: You can add some info that backdooring reduces this penalty

Comment (ashtongreen) [reply doc/source/basicgameplay/companies.rst:18]: Does that belong in the documentation, or is that something better left vague and found by the players?

Comment (phyzical) [reply doc/source/basicgameplay/companies.rst:18]: TIL

**Review by phyzical — APPROVED** [review]
LGTM

File: doc/source/basicgameplay/companies.rst (hunk 3 above)
Comment (ashtongreen) [reply doc/source/basicgameplay/companies.rst:18]: > You can add some info that backdooring reduces this penalty

I'm not sure if that's public knowledge or more discovered knowledge. 

I vaguely hinted at that in the backdoor changes in the hacking documentation.

Comment (ApamNapat) [reply doc/source/basicgameplay/companies.rst:18]: When I found out I kinda felt like - I would have liked to have read that in the docs - but I get your point

Comment (ashtongreen) [reply doc/source/basicgameplay/companies.rst:18]: In #3010, I mention that backdoors reduce penalties. So it's in there now.

**Pull request merged**